Document how to connect to unmanaged resources (#7965)
elastic/cloud-on-k8s: docs/orchestrating-elastic-stack-applications/connect-to-unmanaged-resources.asciidoc

:page_id: connect-to-unmanaged-resources
ifdef::env-github[]
****
link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
****
endif::[]
[id="{p}-{page_id}"]
= Connect to external Elastic resources

> Fields like `elasticsearchRef` or `kibanaRef` are useful to automatically establish connections between applications managed by a same operator instance. It is however also possible to provide existing configuration settings and credentials in a `Secret` through the `secretName` attribute:

Reviewer: ```suggestion
Fields like `elasticsearchRef` or `kibanaRef` are useful to automatically establish connections between applications managed by the same ECK operator instance. It is however also possible to connecto to applications managed by a different ECK operator instance or to applications not managed by ECK, for example an Elastic Cloud deployment.  This can be done by providing connection details and credentials in a `Secret` through the `secretName` attribute:
```

Not sure about the exact wording but I wanted to highlight the use cases for external connections a bit more.

[source,yaml,subs="attributes"]
----
apiVersion: v1
kind: Secret
metadata:

>   name: external-es-ref3

Reviewer: ```suggestion
  name: external-es-ref
```

Author: Good catch, leftover from my tests...

stringData:
  url: https://sample.gcp.elastic-cloud.com
  username: "elastic"
  password: REDACTED
---
apiVersion: kibana.k8s.elastic.co/v1
kind: Kibana
metadata:
  name: kibana-sample
spec:
  version: 8.14.0
  count: 1
  elasticsearchRef:
    secretName: external-es-ref
----

> In the case Agent you can also specify several named references:

Reviewer: ```suggestion
In the case of Elastic Agent you can also specify several named references:
```
